src/benchmark_runner.cc

```diff
@@ -194,27 +199,23 @@ BenchTimeType ParseBenchMinTime(const std::string& value) {
   return ret;
 }
 
-double GetMinTime(const benchmark::internal::BenchmarkInstance& b) {
+double GetMinTime(const benchmark::internal::BenchmarkInstance& b,
+                  const BenchTimeType& iters_or_time) {
   if (!IsZero(b.min_time())) return b.min_time();
-  BenchTimeType iters_or_time = ParseBenchMinTime(FLAGS_benchmark_min_time);
   // If the flag was used to specify number of iters, then return 0 for time.
   if (iters_or_time.tag == BenchTimeType::ITERS) return 0;
```

Comment:
the current default minimum time is 0.5. shouldn't we keep that?

Reply:
Ah, good point